firewall-examples.md

```diff
@@ -15,28 +13,23 @@ $ sdc-enablemachinefirewall &lt;machine-id&gt;
 </div></div>
 
 
-<p>Recall that for instances that have Cloud Firewall feature enabled, the default policy is always:</p>
+Recall that for instances that have Cloud Firewall feature enabled, the default policy is always:
 
-<ul>
-	<li>allow incoming ping requests</li>
-	<li>block all other incoming traffic</li>
-	<li>allow all outgoing traffic</li>
-</ul>
+	- allow incoming ping requests
+	- block all other incoming traffic
+	- allow all outgoing traffic
 
+The examples here are not mutually exclusive. You can combine the rules in the examples as you see fit.
 
-<p>The examples here are not mutually exclusive. You can combine the rules in the examples as you see fit.</p>
+    - [Allow SSH Traffic](CloudFirewallExamples-AllowSSHTraffic)
+    - [Allow HTTP Traffic](CloudFirewallExamples-AllowHTTPTraffic)
+    - [Multiple Web and Database Server Setup](CloudFirewallExamples-MultipleWebandDatabaseServerSetup)
+    - [Bastion Host Setup](CloudFirewallExamples-BastionHostSetup)
 
-<div>
-<ul>
-    <li><a href='#CloudFirewallExamples-AllowSSHTraffic'>Allow SSH Traffic</a></li>
-    <li><a href='#CloudFirewallExamples-AllowHTTPTraffic'>Allow HTTP Traffic</a></li>
-    <li><a href='#CloudFirewallExamples-MultipleWebandDatabaseServerSetup'>Multiple Web and Database Server Setup</a></li>
-    <li><a href='#CloudFirewallExamples-BastionHostSetup'>Bastion Host Setup</a></li>
-</ul></div>
 
-<h1><a name="CloudFirewallExamples-AllowSSHTraffic"></a>Allow SSH Traffic</h1>
+#[Allow SSH Traffic](CloudFirewallExamples-AllowSSHTraffic)
```

Comment:
Put a space between the `#` and the heading text to make it format correctly.

Extra: It's likely these should be `##` or `###` headings for better SEO. Only the page title should be `#`.